### PR TITLE
update sentinel signer expiry check

### LINKED ISSUE #1741: Update sentinel check for signer expiry
Currently sentinel fetches signer addresses and expiry dates from orakl node and for KF node those values are hardcoded in the sentinel. After Nick's implementation of auto signer renewal, we want the sentinel to fetch addresses and expiry dates from the SubmissionProxy:

Note: one SubmissionProxy stores addresses and expiry dates for both Orakl and KF nodes

### PR DESCRIPTION
**Pull request by Intizar-T**
# Description

Previously:
 - get orakl signer address from node endpoint
 - use hardcoded kf signer address
 - send slack message if it's 5 weeks left until either of the signers expire, every 6 hours

Currently:
 - get orakl and kf signer addresses and expiry dates from SubmissionProxy
 - check if any of the signers have expired every 24 hours and send slack message 

Fixes #1741 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [x] Should publish Docker image

### CODE REVIEW

File: sentinel/pkg/checker/signer/app.go [hunk 1]
```diff
@@ -20,91 +18,81 @@ type RegisteredSigner struct {
...
+		log.Error().Err(err).Msg("Failed to get signer addresses")
+		return err
+	}
+
+	for _, signerAddress := range signerAddresses {
+		exp, err := ExtractExpirationFromContract(ctx, jsonRpcUrl, submissionProxyContractAddr, signerAddress)
+		log.Info().Msgf("Signer: %s, Expiration: %s", signerAddress, (*exp).String())
+		if err != nil || exp == nil {
+			log.Error().Err(err).Msg(fmt.Sprintf("Failed to extract expiration for signer: %s", signerAddress))
+			alarmMessage += fmt.Sprintf("Failed to extract expiration for signer: %s\n", signerAddress)
```
Comment (nick-bisonai) [inline sentinel/pkg/checker/signer/app.go:86]: I think it'd be nice if the err message is included in slack message

File: sentinel/pkg/checker/signer/utils.go [hunk 2]
```diff
@@ -34,6 +34,30 @@ func ExtractExpirationFromContract(ctx context.Context, jsonrpc string, submissi
 	return &expirationDate, nil
 }
 
+func GetSignerAddresses(ctx context.Context, jsonrpc string, submissionProxy string) ([]string, error) {
```
Comment (nick-bisonai) [inline sentinel/pkg/checker/signer/utils.go:37]: have you tried to call this function separately? if you didn't I recommend you to try test call this function from separate script

**Review by nick-bisonai — APPROVED** [review]
lgtm! left few comments

File: sentinel/pkg/checker/signer/app.go (hunk 1 above)
Comment (Intizar-T) [reply sentinel/pkg/checker/signer/app.go]: done!

**Pull request merged**